include/SugarObjects/LanguageManager.php

```diff
@@ -202,6 +202,11 @@ public static function refreshLanguage($module, $lang, $loaded_mod_strings = arr
                     'custom/modules/'.$module.'/Ext/Language/'.$lang.'.lang.ext.php',
                  );
 
+        require_once('ModuleInstall/ModuleInstaller.php');
+        $mi = new ModuleInstaller();
+        $mi->modules = array($module);
+        $mi->merge_files('Ext/Language/', $lang . '.lang.ext.php', $lang);
```

Comment:
Would this not have worrying security implications? I could pass in a local or remote PHP file and merge it with the ```lang.ext.php```.

Reply:
Not exactly sure how this could be exploited.

Reply:
Debugged this further and couldn't find any obvious place that this could be injected. LGTM.